lib/google_search_data_viewer_worker/keywords/keyword_search_worker.ex

```diff
@@ -0,0 +1,35 @@
+defmodule GoogleSearchDataViewerWorker.Keywords.KeywordSearchWorker do
+  use Oban.Worker,
+    queue: :keyword_search,
+    max_attempts: 3,
+    unique: [period: 30]
+
+  alias GoogleSearchDataViewer.Keywords.GoogleSearchClient
+  alias GoogleSearchDataViewer.Keywords.Keyword
+
+  @max_attempts 3
+
+  @impl Oban.Worker
+  def perform(%Oban.Job{args: %{"keyword_id" => keyword_id}, attempt: @max_attempts}) do
+    {:ok, _} =
+      keyword_id
+      |> Keyword.get_keyword_upload()
+      |> Keyword.update_keyword_upload_status(:failed)
+
+    {:error, "max attempts reached, attempt: #{@max_attempts}"}
+  end
+
+  @impl Oban.Worker
```

Comment:
I think this can be remove because we already declare in the above function clause

Reply:
The   `@impl Oban.Worker`, right? Alright I will remove.

I thought it would make it even more confusing if only one of the `perform` functions has this - ie you may wonder if the other function is from `keyword_search_worker` or `Oban.Worker`